src/main/webapp/i18n/de/health.json

```diff
@@ -17,7 +17,9 @@
             "hystrix": "Hystrix",
             "diskSpace": "Festplattenspeicher",
             "mail": "Email",
-            "db": "Datenbank"
+            "db": "Datenbank",
+            "bamboo": "Bamboo",
```

Comment:
What would happen if the server would respond a different CI and VCS system?
I think we could make this already a bit generic here

Reply:
I now use generice CI and VCS names